alpaca/data/models/base.py

```diff
@@ -37,6 +38,7 @@ class BaseDataSet(BaseModel):
     """
 
     data: Dict[str, List[BaseModel]]
+    model_config = ConfigDict(protected_namespaces=())
```

Comment:
I would prefer `tuple()` to just `()`, looks more pythonic and readable to me. Also if there was a way to let pydantic models inherit the ConfigDict from the BaseDataSet model we could avoid some duplication.

Reply:
Understood. Addressed https://github.com/alpacahq/alpaca-py/pull/353/commits/76daf21d70545dadbf77bdc88a437630d5869553